Projects/Foundations/Schemas/Docs-Topics/O/Omega/Omega-One-Data-Mine/omega-one-data-mine-001-omega-one-overview.json

```diff
@@ -3,21 +3,42 @@
     "pageNumber": "1",
     "type": "Omega One - Overview",
     "definition": {
-        "text": "You can set different values for used periods at the top section of most indicators found in the Omega Data Mine -> Data Building Procedure -> Procedure Loop -> Procedure Javascript Code node."
+        "text": "You can set different values for used periods at the top section of most indicators found in the Omega Data Mine -> Data Building Procedure -> Procedure Loop -> Procedure Javascript Code node.",
+        "translations": [
```

Comment:
@harrellbm Isn`t here an: "updated" missing? Iam not sure but in other pull requests there is an: "updated" between text and translations like in this file for example: Projects/Foundations/Schemas/Docs-Topics/N/Nodes/Nodes/nodes-008-node-attaching-rules.json

Reply:
The field 'updated' appears after editing the English text, if there was no editing, then this field will not be.

Reply:
That is true. This is one of the brand new docs created by quantum8 last
Month so it had not been edited at all so the updated field has not been added.  As soon as there are any updates to it then the system will add it